aten/src/ATen/native/ConvUtils.h

```diff
@@ -83,12 +84,12 @@ static inline Tensor reshape_bias(int64_t dim, const Tensor& bias) {
 
 static inline bool cudnn_conv_use_channels_last(const at::Tensor& input, const at::Tensor& weight) {
   // disable NHWC for float64 input.
-  if (!detail::getCUDAHooks().compiledWithCuDNN() ||
+  if (!at::detail::getCUDAHooks().compiledWithCuDNN() ||
```

Comment:
Without this I get compilation errors because it's looking in the wrong namespace:
```
/home/peter/git/pytorch/aten/src/ATen/native/ConvUtils.h(87): error: namespace "at::native::detail" has no member "getCUDAHooks"
```